<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>object</class>
 <widget class="QDialog" name="object">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QProgressBar" name="progress_bar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>640</y>
     <width>851</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="download_button">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>670</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quit_button">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>670</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter your access Token</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>641</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Select the directory where the files you wish to download are stored.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>641</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter the directory to which you wish to save the files.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>54</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="token">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>851</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="local_savepoint">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>761</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="abort_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>670</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Abort</string>
   </property>
  </widget>
  <widget class="QPushButton" name="folder_button">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>250</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Folder</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_files_button">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>590</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="file_list">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>849</width>
     <height>276</height>
    </rect>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>861</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>List all of the files you wish to download below. Each file needs to be entered on a separate row</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="goholm_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>96</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>GÃ¶holm</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="ottenby_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>96</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ottenby</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="other_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>96</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Other</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="undoRedoEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>453</x>
     <y>100</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Year:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>100</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
